src/main/java/be/syntra/devshop/DevshopFront/services/utils/ProductMapperUtil.java

```diff
@@ -22,4 +31,38 @@ public ProductDto convertToProductDto(Product product) {
                 .categoryNames(categoryMapperUtil.mapToCategoryNames(product.getCategories()))
                 .build();
     }
+
+    public Product convertToProduct(ProductDto productDto) {
+        return Product.builder()
+                .id(productDto.getId())
+                .name(productDto.getName())
+                .price(productDto.getPrice())
+                .description(productDto.getDescription())
+                .archived(productDto.isArchived())
+                .totalInCart(0)
+                .categories(categoryMapperUtil.mapToCategories(productDto.getCategoryNames()))
+                .build();
+
+    }
+
+    public ProductDtoList convertToProductDtoList(ProductListDto productListDto) {
+        List<ProductDto> productDtoList = productListDto.getProducts().stream()
+                .map(this::convertToProductDto)
+                .map(this::setProductCountInProductDto)
+                .collect(Collectors.toList());
+        return new ProductDtoList(productDtoList);
+    }
+
+    private ProductDto setProductCountInProductDto(ProductDto productDto) {
+        productDto.setTotalInCart(getCountFormProductInCart(productDto.getId()));
+        return productDto;
+    }
+
+    private int getCountFormProductInCart(Long id) {
```

Comment:
Possible typo in method name getCount**Form**...

Reply:
fixed